StepLang.Tests/Framework/IO/LengthFunctionTest.cs

```diff
@@ -0,0 +1,80 @@
+using System.Diagnostics.CodeAnalysis;
+using StepLang.Framework.Conversion;
+using StepLang.Interpreting;
+using StepLang.Parsing.Expressions;
+
+namespace StepLang.Tests.Framework.IO;
+
+public class LengthFunctionTest
```

Comment:
Can you add at least one test case using variable expressions? I think variable expressions are the most common use case for this function.